src/feeds/music-trending.feed.ts

```diff
@@ -0,0 +1,39 @@
+import { Feed } from '../core/feed';
+import { MusicTrendingFeedResponseItemsItem, MusicTrendingFeedResponseRootObject } from '../responses';
+import { IgAppModule } from '../types/common.types';
+
+export class MusicTrendingFeed extends Feed<MusicTrendingFeedResponseRootObject, MusicTrendingFeedResponseItemsItem> {
+  protected nextCursor?: string;
+  protected moreAvailable: boolean = true;
+
+  public product: IgAppModule;
+
+  async items(): Promise<MusicTrendingFeedResponseItemsItem[]> {
+    const response = await this.request();
+    return response.items;
+  }
+
+  async request(): Promise<MusicTrendingFeedResponseRootObject> {
+    const { body } = await this.client.request.send<MusicTrendingFeedResponseRootObject>({
+      url: '/api/v1/music/trending/',
+      method: 'POST',
+      form: {
+        cursor: this.nextCursor || '0',
+        _csrftoken: this.client.state.cookieCsrfToken,
+        product: this.product,
+        _uuid: this.client.state.uuid,
+        browse_session_id: this.client.state.clientSessionId,
+      },
+    });
```

Comment:
Looks like you've forgot to in all the feeds in your PR
```ts
this.state = body;
```
To be honest i don't like this type of setting state - too easy to forgot it. Do you now how to refactor it?

Reply:
The only option that comes to my mind, would be to have a protected method `send(options)`, which accepts the same options as `this.client.request.send()` does and every feed has to implement the `send()` instead. (the send method can then set the state)